src/main/java/org/gridsuite/modification/utils/ModificationUtils.java

```diff
@@ -1527,21 +1526,18 @@ public static Map<UUID, FilterEquipments> getUuidFilterEquipmentsMap(IFilterServ
         return isValidFilter ? exportFilters : null;
     }
 
-    public static Map<UUID, FilterEquipments> getUuidFilterWrongEquipmentsIdsMap(ReportNode subReportNode, Map<UUID, FilterEquipments> exportFilters, Map<UUID, String> filters) {
+    public static void logFilterWrongEquipmentsIds(ReportNode subReportNode, Map<UUID, FilterEquipments> exportFilters, Map<UUID, String> filters) {
```

Comment:
I'd rather this but not important at all (the remove 's' is on purpose) : 
```suggestion
    public static void logWrongEquipmentIdsInFilters(ReportNode subReportNode, Map<UUID, FilterEquipments> exportFilters, Map<UUID, String> filters) {
```